js/vcc.js

```diff
@@ -255,6 +255,8 @@ module.exports = class vcc extends Exchange {
                 'code': code,
                 'name': this.safeString (currency, 'name'),
                 'active': active,
+                'deposit': canDeposit,
+                'withdraw': canWithdraw,
```

Comment:
The values of `deposit` and `withdraw` should be `true` or `false`, but not `1`.